export/cmake/CMakeLists.txt.export

```diff
@@ -194,7 +194,7 @@ endif()
 # shared and static libraries built from the same object files
 if (LIBINT2_BUILD_SHARED_AND_STATIC_LIBS)
   add_library(libint2 SHARED $<TARGET_OBJECTS:libint2_obj>)
-  set_target_properties(libint2 PROPERTIES LIBRARY_OUTPUT_NAME int2)
+  set_target_properties(libint2 PROPERTIES LIBRARY_OUTPUT_NAME int2 VERSION 2.0.4 SOVERSION 2)
```

Comment:
would `${LIBINT_VERSION}` would for `VERSION`?

Reply:
As the API/ABI are not strictly controlled for backwards compatibility, the SOVERSION should be the same as the VERSION (better safe than sorry ...). This is the CMake default if SOVERSION is omitted.

```
set_target_properties(libint2 PROPERTIES
    LIBRARY_OUTPUT_NAME int2
    VERSION ${LIBINT_VERSION}
)
```

Reply:
Thanks. I think `SOVERSION 2` is appropriate, the ABI of the library has been stable for many years.